data_collection/gazette/spiders/ba/ba_maragogipe.py

```diff
@@ -0,0 +1,11 @@
+import datetime as dt
+
+from gazette.spiders.base.sai import SaiGazeteSpider
+
+
+class BaMaragogipeSpider(SaiGazeteSpider):
+    TERRITORY_ID = "2920601"
+    name = "ba_maragojipe"
+    state_city_url_part = "ba/maragojipe"
+    start_date = dt.date(2023, 1, 1)
```

Comment:
```suggestion
    start_date = dt.date(2011, 2, 2)
```
A página tem um filtro por ano que vai até 2011. Não começa em 2023.